TypeCobol/Compiler/Nodes/Attributes.cs

```diff
@@ -267,6 +291,40 @@ public object GetValue(object o, SymbolTable table)
             throw new System.ArgumentOutOfRangeException("Too many receiving items (" + variablesWritten.Count + ")");
         }
     }
+    internal class PointerDisplayableReceiversAttribute : Attribute
+    {
+        public object GetValue(object o, SymbolTable table)
+        {
+            var node = (Node)o;
+            var codeElement = node.CodeElement;
+            var setStatement = codeElement as SetStatementForIndexes;
+            List<string> displayableWritten = new List<string>();
+            List<DataDefinition> dataDefWritten = new List<DataDefinition>();
+            if (setStatement != null && node.IsFlagSet(Node.Flag.NodeContainsPointer))
+            {
+                foreach (var data in setStatement.StorageAreaWrites)
+                {
+                    //TODO: handle the case of ambiguous variable
+                    DataDescription dataDef = node.SymbolTable.GetVariablesExplicitWithQualifiedName(data.MainSymbolReference.URI)
```

Comment:
See with what Cosmin did, we now shloud be able to get this information without calling SymbolTable.